draft-ietf-netmod-schedule-yang.md

```diff
@@ -102,6 +102,39 @@ Also, this document uses the YANG terminology defined in {{Section 3 of !RFC7950
 
 #  Module Overview
 
+##  Features and Augmentations
+
+   The "ietf-schedule" data model defines the recurrence related groupings using
+   a modular approach. Basic, intermediate, and advanced representation of recurrence
+   groupings are defined, with each reusing the previous one and adding more parameters.
+   To allow for different options, two features are defined in the data model:
+
+   *  "basic-recurrence-supported"
+   *  "icalendar-recurrence-supported"
+
+   {{features}} provides an example about how these features can be used. Implementations
+   may support a basic recurrence rule or an advanced one as needed, by declaring
+   different features. Whether only one or both features are supported is implementation
+   specific and depend on specific scheduling context.
+
+   The groupings defined in the document can also be augmented to support specific needs. As an example,
+   {{augments}} demonstrates how additional parameters can be added to comply with specifc schedule needs.
+
+  ##  Types and Identities
+
+   The "ietf-schedule" module (Section 7) defines the following identities:
```

Comment:
```suggestion
   The "ietf-schedule" module ({{sec-schedule}}) defines the following identities:
```

Reply:
I am thinking whether it would be better for features, types and identities come first as a subsection, and future augmentation separated as the section after module overview (augmentation itself doesn't seem like a part of the module).

Reply:
Looks good to me. Please check the updated text. Thanks